src/stream.jl

```diff
@@ -595,9 +595,12 @@ function stats(stream::TranscodingStream)
         out = transcoded_out - buffersize(buffer1)
     elseif mode === :write
         transcoded_in = buffer1.transcoded
-        transcoded_out = buffer2.transcoded
```

Comment:
`buffer2` is "owned" by the underlying stream if it is shared, so this stream shouldn't rely on `buffer2.transcoding` being the expected value.